src/transaction/consensus_parameters.rs

```diff
@@ -420,4 +478,5 @@ pub mod default_parameters {
     pub const MAX_PREDICATE_DATA_LENGTH: u64 =
         ConsensusParameters::DEFAULT.max_predicate_data_length;
     pub const GAS_PRICE_FACTOR: u64 = ConsensusParameters::DEFAULT.gas_price_factor;
+    pub const MAX_MESSAGE_DATA_LENGTH: u64 = ConsensusParameters::DEFAULT.max_message_data_length;
```

Comment:
Since these are deprecated, we probably don't want to add new constants here?

Reply:
Why is message data deprecated?

Reply:
Its important we always keep consistency. If we are to remove these constants, we should remove all of them in one release. The user will still receive the deprecated warning tho.

@adlerjohn we are using the `ConsensusParameter` struct to fetch these values. These constants will be removed soon.